### DISCUSSION TITLE
Increase Mutating webhook timeout.

### DISCUSSION THREAD

**Discussion by smithdebug** [linkerd/linkerd2#11984] [Help]
Hello Team,
I’m reaching out to inquire about the possibility of extending the timeout for the `linkerd-proxy-injector-webhook-config` Mutating Webhook. Currently, the timeout is set to `10 seconds` bydefault. We’ve recently faced a situation where this timeout caused an issue, resulting in a Pod without a Linkerd proxy sidecar. Thanks

**Comment by alpeb** [comment]
This sounds totally reasonable to me. Moving into an issue.

**Reply by mikebell90** [reply]
@smithdebug I just submitted a PR for this, but really you should set the webhookFailurePolicy to fail. That way no pod can come up without the webhook being successful

**Reply by smithdebug** [reply]
Thanks, @mikebell90 . Yes i have set the webhookFailurePolicy to fail now, We had missed that earlier and was set to Ignore. Thanks
